### ISSUE TITLE
MaterialstoreException: the left material is not an image ...

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/materialstore#362]
derived from the post of VisualInspectionInkatalonStudio_Reborn project

- https://github.com/kazurayam/VisualInspectionInKatalonStudio_Reborn/issues/25

```
Reason:
com.kazurayam.materialstore.core.filesystem.MaterialstoreException: the left material is not an image: {"jobName":"_","jobTimestamp":"_","id":"0000000000000000000000000000000000000000","fileType":"","metadata":{},"identification":"{}","relativeUrl":"_/_/objects/0000000000000000000000000000000000000000.","diffability":"UNABLE"}
	at com.kazurayam.materialstore.base.reduce.differ.ImageDifferToPNG.stuffDiff(ImageDifferToPNG.java:40)
	at com.kazurayam.materialstore.base.reduce.DiffingMPGProcessor.stuffDiffByDiffer(DiffingMPGProcessor.java:68)
...
```

How can I manage a case where the right MaterialList had a screenshot taken from the URL `http://demoaut-mimic.kazurayam.com/` and the program can not find a screenshot of a URL with path `/` in the left MaterialList: in fact the left MaterialList had a screenshot taken from the URL `http://demoaut-mimic.katalon.com/index.html`. These 2 URLs are slightly differnet. Man can think of these 2 URLs make a pair; but my program can not.

**Comment by kazurayam** [comment]
My program should not throw the Exception to stop the processing entirely.

My program should show a WANING message, but continue the processing.

When "the left material is not an image", then my program can substitute the left material a blank image and create a diff, which will be very read. This would tell us that there is a problem here. That's OK.

**Comment by kazurayam** [comment]
resolved at 0.12.5-SNAPSHOT

<img width="1072" alt="スクリーンショット 2022-11-18 11 07 12" src="https://user-images.githubusercontent.com/4467417/202600506-4ab64133-9a46-40f4-8e89-fba5bb4429d4.png">

<img width="1072" alt="スクリーンショット 2022-11-18 11 06 54" src="https://user-images.githubusercontent.com/4467417/202600523-fb5057bd-2ee5-4675-a64f-6065163bf653.png">

<img width="1072" alt="スクリーンショット 2022-11-18 11 07 03" src="https://user-images.githubusercontent.com/4467417/202600533-47230045-8d11-4176-bd50-66971c68f223.png">

**Comment by kazurayam** [comment]
fixed at the 0.12.5
